urbanopt/geojson-modelica-translator
# Use mblv9

#### Any background context you want to provide?
The GMT should work with mainline MBL, to enable new capabilities as they are developed

_Until we have a compiler for MSLv4, success for this PR will be defined as:_
- [x] the manual test procedure working no matter which model you open
- [x]  all other pytests passing (as of 2022-05-11 this is 18 failing, 78 passing)
#### What does this PR accomplish?
Update templates, code, and test files to work with MBLv9
#### How should this be manually tested?
- Check out MBL master branch
- `poetry run py.test`
- open a model in Dymola
- Simulate the model
- Bask in the glow of a simulated model without errors 🎉 
#### What are the relevant tickets?
#409
